samples/tls/path-based-ingress/monitoring/user-values-prom-operator.yaml

```diff
@@ -50,6 +50,14 @@ grafana:
 #   prometheusSpec:
 #     routePrefix: /prometheus
 #     externalUrl: https://host.cluster.example.com/prometheus
+#     alertingEndpoints:
```

Comment:
Now that I think about it, if a user had `TLS_ENABLE='true'` with `MON_TLS_PATH_INGRESS='true'` and did and update, without updating their configuration, prometheus alerts would not make their way to alertmanager. This section would need to be included. 

Thoughts?